### ISSUE TITLE
fullName not used in UserDetails constructor for auto-create accounts

### ISSUE THREAD

**Issue by 68lc040** [bitium/jira-saml-plugin#59]
Hey guys on line 64 in SsoJiraLoginServlet.java, as of this submission, the UserDetails constructor is passed 'username' for both arguments whereas presumably, based on the preceding lines, 'username' and 'fullName' was the intention?  From Atlassian's javadoc:

Constructor Detail

    UserDetails

    public UserDetails(String username,
                       String displayName)

The effect is that currently there is no use in asserting the 'cn' attribute as it is not used, meaning that the 'Full name' field in the jira account will reflect the data in the 'Username' field when auto creating accounts.  I'd be happy to submit a PR for this, whatever works for y'all.  Thanks for the free plugin, much appreciated.

**Comment by cbanack** [comment]
Yeah, we've noticed this problem too.    Just a typo, by the looks of it,
